PendingReleaseNotes.md

```diff
@@ -13,6 +13,9 @@ v1.5...
 - Ceph mons require an odd number for a healthy quorum. An even number of mons is now disallowed.
 - Update deprecated CRD apiextensions.k8s.io/v1beta1 to v1 ([#6424](https://github.com/rook/rook/pull/6424))
 - preservePoolsOnDelete is deprecated for CephFilesystem and is no longer allowed because of data-loss concerns. preserveFilesystemOnDelete acts as a replacement and preserves the filesystem when the CephFilesystem CRD is deleted (which implicitly includes all associated pools). See [#6495](https://github.com/rook/rook/pull/6495) for more details.
+- The discovery daemon is disabled by default since the discovery is not necessary in most clusters. Enable the discovery daemon if:
+   - If devices are being added to nodes and you want to automatically configure OSDs on new devices without restarting the operator.
+   - If the configmaps with device information is expected to be discovered.
```

Comment:
I think this statement will be a little too vague for anyone except really knowledgeable and advanced Rook users. Could we say "If the user wishes to inspect the ConfigMaps which are loaded with host device information"?

Reply:
I think I'll just drop this bullet to avoid confusion since I doubt hardly anyone is actually consuming these configmaps.